components/aws/sagemaker/tests/integration_tests/component_tests/test_train_component.py

```diff
@@ -78,4 +79,9 @@ def test_trainingjob(
     else:
         assert f"dkr.ecr.{region}.amazonaws.com" in training_image
 
+    error_message = 'Error in fetching CloudWatch logs for SageMaker job'
```

Comment:
Can we reference the message string as a variable? Perhaps turn it into a constant in the other file. This magic string would require we update each file if the message ever changed.